jdk/src/solaris/bin/java_md_solinux.c

```diff
@@ -380,6 +380,11 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                                            asked for? Current model is
                                            fine unless another model
                                            is asked for */
+#ifdef AIX
+    static char *mallocOptins = "MALLOCOPTIONS=multiheap,buckets";
+    if (putenv(mallocOptins) != 0)
```

Comment:
Pls use { }

Reply:
This overwrites whatever is set in the environment so a user cannot have custom settings, I don't think it should be done this way.